pwa/src/services/getNameFromBijlageArray.ts

```diff
@@ -0,0 +1,8 @@
+export const getNameFromBijlageArray = (url: string, array: any[]) => {

```

Comment:
I think we should name this `array` param, e.g. `bijlagen: any[]`?

Reply:
Maybe `getBijlageTitleFromURL` would be a more descriptive name for this function.